recipes/catch2/2.x.x/conanfile.py

```diff
@@ -14,8 +14,16 @@ class ConanRecipe(ConanFile):
     exports_sources = ["CMakeLists.txt", "patches/**"]
     generators = "cmake"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"fPIC": [True, False], "with_main": [True, False]}
-    default_options = {"fPIC": True, "with_main": False}
+    options = {
+        "fPIC": [True, False],
+        "with_main": [True, False],
+        "with_benchmark": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "with_main": False,
+        "with_benchmark": False,
+    }
     _cmake = None
 
```

Comment:
```suggestion
    def validate(self):
        if tools.Version(self.version) < "2.21.1" and self.settings.arch == "armv8":
            raise ConanInvalidConfiguration("ARMv8 is supported by 2.21.1+ only! give up!")
```

it seems like old versions of `catch` do not properly support ARM, e.g. they use x86 assembly:
```
#define CATCH_TRAP() __asm__("int $3\n" : : ) /* NOLINT */
```

Reply:
@SSE4 do you mean 2.12.4? I dont see a 2.21.x tag.

Reply:
yes. 2.12, not 2.21. sorry for the typo.